db/migrate/20160517224513_add_null_false_to_auction_fields.rb

```diff
@@ -0,0 +1,32 @@
+class AddNullFalseToAuctionFields < ActiveRecord::Migration
+  def up
+    change_column :auctions, :billable_to, :string, default: ''
```

Comment:
explanation of this: when fields are able to be null, I prefer to set the default to empty string at the DB level. That way, no `nil` errors when we render the obj in the view. :)